### PR TITLE
[DB-182] [risk=no] Update concept charts as per new design

### PR DESCRIPTION
**Pull request by SrushtiGangireddy**
1. Add tabs for biological sex/ gender identity/ age and sources charts as per new designs.

### CODE REVIEW

File: public-ui/src/app/data-browser/concept-charts/concept-charts.component.ts [hunk 1]
```diff
@@ -18,7 +18,8 @@ export class ConceptChartsComponent implements OnChanges, OnInit, OnDestroy {
   @Input() concept: Concept;
   @Input() backgroundColor = '#ECF1F4'; // background color to pass to the chart component
   @Input() showSources = true;
-  @Input() showGenderGraph = false;
```
Comment (calbach) [inline public-ui/src/app/data-browser/concept-charts/concept-charts.component.ts]: Doesn't necessarily need to be in this PR, but would it ever make sense to show multiple of these at the same time? If not, a single property with an enum value e.g. `GraphType` would probably make way more sense here and could simplify a lot of this verbose code, e.g. ngSwitch statement.

File: public-ui/src/app/views/ehr-view/ehr-view.component.ts [hunk 2]
```diff
@@ -228,7 +232,11 @@ export class EhrViewComponent implements OnInit, OnDestroy {
       return;
     }
     this.resetSelectedGraphs();
-    this.showGenderGraph = true;
+    if (this.ehrDomain.name === 'Measurements') {
+      this.showMeasurementGenderBins = true;
```
Comment (calbach) [inline public-ui/src/app/views/ehr-view/ehr-view.component.ts]: Why is this case treated differently? Comment?

Comment (SrushtiGangireddy) [reply public-ui/src/app/views/ehr-view/ehr-view.component.ts]: Done.

File: public-ui/src/app/data-browser/concept-charts/concept-charts.component.ts (hunk 1 above)
Comment (SrushtiGangireddy) [reply public-ui/src/app/data-browser/concept-charts/concept-charts.component.ts]: Added enum and made changes. Thanks

File: public-ui/src/app/utils/graphtypes.ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
+export enum graphType {
```
Comment (calbach) [inline public-ui/src/app/utils/graphtypes.ts]: Should be capitalized: `GraphType`

File: public-ui/src/app/views/ehr-view/ehr-view.component.ts [hunk 4]
```diff
@@ -203,45 +199,60 @@ export class EhrViewComponent implements OnInit, OnDestroy {
...
     this.resetSelectedGraphs();
-    if (g === 'Gender') {
-      this.showGenderGraph = true;
-    } else if (g === 'Age') {
-      this.showAgeGraph = true;
-    } else if (g === 'Sources') {
-      this.showSourcesGraph = true;
-    } else {
-      this.showAgeGraph = true;
+    switch (g) {
```
Comment (calbach) [inline public-ui/src/app/views/ehr-view/ehr-view.component.ts]: `this.graphToShow = g` ?

**Comment by SrushtiGangireddy** [comment]
Addressed comments. Thanks. Merging in.

**Pull request merged**